eng/pipelines/templates/stages/platform-matrix-cryptography-dependency.json

```diff
@@ -11,12 +11,23 @@
     "CoverageArg": "--disablecov"
   },
   "include": [
+    {
+      "Config": {
+        "Ubuntu2004_312": {
+          "OSVmImage": "MMSUbuntu20.04",
+          "Pool": "azsdk-pool-mms-ubuntu-2004-general",
+          "PythonVersion": "3.12.0",
+          "CoverageArg": "--disablecov",
+          "TestSamples": "false"
+        }
+      }
+    },
```

Comment:
I think this could be rolled in to the below entry instead:

```
    {
      "Agent": {
        "Ubuntu2004": {
          "OSVmImage": "MMSUbuntu20.04",
          "Pool": "azsdk-pool-mms-ubuntu-2004-general"
        },
        "PythonVersion": ["3.11", "3.12.0"],
        "CoverageArg": "--disablecov",
        "TestSamples": "false"
      }
    },
```

Reply:
I like it. Will do.